recipes/sfepy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set rev = "d479f7a9" %}
+{% set name = "sfepy" %}
+{% set org = "sfepy" %}
+{% set version = "d479f7a9" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/{{ org }}/{{ name }}.git
+  git_rev: {{ rev }}
+
+build:
+  number: 0
+  script: CFLAGS='-I${PREFIX}/include/python${CONDA_PY}m' python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - anaconda
```

Comment:
Anaconda is too generic of a build (or run) requirement. Please narrow the list to the packages that are actually needed to run and build this library.